### PR TITLE
Trivial updates

### PR DESCRIPTION
**Pull request by edburns**
modified:   README.md

- This is a Spring app first and a Java app second. Let's just call it a Spring app.

modified:   infra/bicep/hooks/build-java.sh
modified:   infra/bicep/hooks/export-env.sh

- Must have changed by doing azd up.

modified:   src/ai-shop-ui/src/App.tsx

- Update URL to github repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Java AI - AI Shop
 
-This is a simple AI Shop application that demonstrates how to use Azure OpenAI with Java to create a sample second hand shop. It uses the multi-modal capabilities of the OpenAI API to generate a description of the product based on an image. Instead of filling out a form, the users can simply upload an image of the product they want to sell and the AI will generate all the necessary information and fill the form for them.
+This is a simple AI Shop application that demonstrates how to use Azure OpenAI with Spring to create a sample second hand shop. It uses the multi-modal capabilities of the OpenAI API to generate a description of the product based on an image. Instead of filling out a form, the users can simply upload an image of the product they want to sell and the AI will generate all the necessary information and fill the form for them.
```
Comment (pmalarme) [inline README.md:3]: I would keep Java as the goal is to add a micro-service in Quarkus

File: infra/bicep/hooks/build-java.sh
Comment (pmalarme) [inline infra/bicep/hooks/build-java.sh:1]: Should not be part of the commit if there is no change.

File: infra/bicep/hooks/export-env.sh
Comment (pmalarme) [inline infra/bicep/hooks/export-env.sh:1]: Should not be part of the commit if there is no change.

**Review by pmalarme — CHANGES_REQUESTED** [review]
Hi Ed,

Thank you for your contribution. Please find some minor comments.

Kind regards,
Pierre

**Comment by SandraAhlgrimm** [comment]
Thanks Ed, UI update is handled here https://github.com/Azure-Samples/java-ai/pull/36.
We'll stay with Java as the sample should receive at least one Quarkus and Semantic Kernel service in future.

**Pull request closed without merging**